modules/acm/feature.tf

```diff
@@ -55,6 +55,7 @@ resource "google_gke_hub_feature_membership" "main" {
 
       content {
         enabled                    = true
+        referential_rules_enabled  = true
```

Comment:
wait, sorry for being late to this, but instead of hard coding this to true, shouldn't we create a variable that can be set but which defaults to `true`?

Reply:
at this point, i guess we can make this suggestion itself a feature request

Reply:
ha, i am late to the party -- this was [observed/suggested](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1373#pullrequestreview-1086219026)

Reply:
Hi Steve - I added the FR here: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/issues/1393